### ISSUE TITLE
Conditionally send cancellation email to consumer when cancelling the whole order

### ISSUE THREAD

**Issue by drummer83** [openfoodfoundation/openfoodnetwork#9053]
## Description
Since implementation of #8766 users can choose if a cancellation email should be sent when an order is being cancelled. However this only applies when the order is cancelled because the last line item is deleted (on order edit page or BOM).

## Expected Behavior
When cancelling the whole order on order edit page, users should be able to choose if a cancellation email should be sent or not.

## Actual Behaviour
When cancelling the whole order on order edit page, users only have to confirm to cancel the order. They cannot choose if a cancellation email should be sent or not.

## Steps to Reproduce
1. As customer: place an order
2. As shop manager: cancel the order on the order edit page admin/orders/Rxyz/edit by clicking the 'cancel' button OR by selecting 'cancel order' from the actions drop down
3. Observe the modal

## Animated Gif/Screenshot
This is where you can cancel the order:
![grafik](https://user-images.githubusercontent.com/1790176/160950789-2900f771-d5ee-47d2-9a57-8d9b32c1ec0c.png)

This is what the modal should look like (with adjusted text: Are you sure you want to cancel the order?):
![grafik](https://user-images.githubusercontent.com/1790176/160950914-c6ac2d0d-2b83-4ff4-a19c-5dddb189bf99.png)

## Workaround
Cancel the order by deleting item by item.

## Severity
bug-s3: a feature is broken but there is a workaround

## Your Environment
* Version used: 4.1.23
* Browser name and version:
* Operating System and version (desktop or mobile):

**Comment by RachL** [comment]
@drummer83 I think this is more an enhancement than a bug? So maybe best to move it to wishlist to see if it is a papercut. What do you think?

**Comment by drummer83** [comment]
Yes, I was unsure about that. Still I would love to see this being prioritized for implementation to have a consistent behavior of this new feature rather letting it sit there for a while with this inconsistency.
Additionally it was called a 'nightmare' by @lin-d-hop and developers and testers have this fresh on their minds. But I know that we are/have to be very strict on processes sometimes... :-)

**Comment by lin-d-hop** [comment]
Best ignore my comments. OFN is a nightmare to me.
